dubbo-compatible/src/license/THIRD-PARTY.properties

```diff
@@ -0,0 +1,34 @@
+# Generated by org.codehaus.mojo.license.AddThirdPartyMojo
```

Comment:
Why do we need to add this file?

Reply:
there are some library don't have license info, so license plugin provide a way called missingFile to provide license information for those library.
If we don't use them, we can create a plain text and replace one by one with script

Reply:
Yes, it is better to put it to an all-in-one file. BTW, bea license is said to be incompatible with Apache. We may need to spot that.